aws/DRVault-Single-Deployment.yaml

```diff
@@ -395,14 +395,30 @@ Resources:
 
           def handler(event, context):
             try:
-              response = boto3.client('ec2').describe_images(
-                  Owners=['138339392836', '284566481789', '923248178732', '604515549189'],
-                  Filters=[
-                    {'Name': 'tag:Cyberark_version', 'Values': [event['ResourceProperties']['Version']]},
-                    {'Name': 'tag:component', 'Values': [event['ResourceProperties']['Component']]},
-                    {'Name': 'tag:os', 'Values': [event['ResourceProperties']['OS']]},
-                  ],
-              )
+              ver= event['ResourceProperties']['Version']
```

Comment:
Not sure `ver` is clear enough here. maybe call it `pasVer` ?

Reply:
done